docs/content/en/tasks/migrate/_index.md

```diff
@@ -6,227 +6,245 @@ weight: 1
+```
 
-{{% alert title="Recommendation" color="success" %}}
-If you change the replication strategy of a keyspace, run a full, cluster-wide repair on it using whatever solution you have for scheduling and running repairs. If you do not already have another solution, you can run `nodetool repair -full` on each Cassandra node, one at a time. Repairs can be both time consuming and resource intensive. It is best to run them during a scheduled maintenance window. 
-{{% /alert %}}
+**Note: Modify the version in the URL above to match your target version.**
 
-## Check the endpoint snitch
+## Copy secrets from K8ssandra 1.x
 
-Make sure that `GossipingPropertyFileSnitch` is used, and not `SimpleSnitch`.
+Secrets can only be referenced in pods if they exist in the same namespace. Therefore when migrating from K8ssandra 1.x, the secrets that were created in the original k8ssandra namespace need to be copied over to the k8ssandra-operator namespace.
 
-{{% alert title="Recommendation" color="success" %}}
-If you change the snitch, run a full, cluster-wide repair.
-{{% /alert %}}
+That includes the following secrets:
 
-## Client changes
+- Superuser
+- Reaper (CQL)
+- Reaper JMX
+- Medusa
+- Medusa bucket key
+- Stargate
 
-Make sure client services are using a `LoadBalancingPolicy` that will route requests to the local datacenter. Also make sure your clients are using `LOCAL_*` and not `QUORUM` consistency levels.  
+There is no automation to copy secrets in kubernetes, so they need to be fully recreated either manually or through CD automation.
 
-Here is an example `application.conf` file for version 4.11 of the Java driver that configures the `LoadBalancingPolicy` and the default consistency level:
+When migrating from Cassandra, check the [security]({{< relref "/tasks/secure" >}}) section of our documentation to read about the expected secrets and create them accordingly.
 
-```conf
-# application.conf
+## Alter your keyspaces replication strategy
 
-datastax-java-driver {
-    basic.load-balancing-policy {
-        local-datacenter = dc1
-    }
+Before spinning up the new datacenter, make sure all user created keyspaces are using `NetworkTopologyStrategy` (NTS). If not, alter them to use NTS and set the same number of replicas as currently on the existing DC:
 
-    basic.request {
-        consistency = LOCAL_QUORUM
-    }
-}
 ```
+ALTER KEYSPACE <my-keyspace> WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1':3};
+```
+
+Alter the following keyspaces in the same way:
+
+- `system_auth`
+- `system_distributed`
+- `system_traces`
+- `data_endpoint_auth` (Stargate)
+- `reaper_db` (Reaper)
+
+
+## Restrict traffic to dc1
 
-## Install K8ssandra
+Client traffic should be restricted to `dc1` by selecting it as local datacenter and using only `LOCAL_` consistency levels (or `QUORUM`). This will ensure consistency during the expansion to a new datacenter.
```

Comment:
Wouldn't QUORUM be dangerous during expansion? People imo should use LOCAL_ONE or LOCAL_QUORUM exclusively.

Reply:
You're right, using LOCAL CLs is safer.